js/demo-app/package-lock.json

```diff
@@ -2626,10 +3493,15 @@
     "iframe-coordinator": {
       "version": "file:../iframe-coordinator",
       "requires": {
+        "@types/jasmine": "2.8.9",
         "elm": "0.19.0-bugfix2",
         "request": "2.88.0"
       },
       "dependencies": {
+        "@types/jasmine": {
```

Comment:
@stormojm ... I went ahead and commited this package-lock.json change that came in from the lib dependency updates.  However, I noticed that these types are listed in dependency.  Could they be moved to dev?